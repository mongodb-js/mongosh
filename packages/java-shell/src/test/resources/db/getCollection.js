// command
db.getCollection('does not exist');
