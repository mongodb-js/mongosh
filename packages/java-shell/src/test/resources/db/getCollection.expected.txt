StringResult: test.does not exist
